main/res/values-zh-rTW/strings.xml

```diff
@@ -482,6 +484,7 @@
     <string name="cache_filter_datefilter_label_from">來自：</string>
     <string name="cache_filter_datefilter_label_to">至:</string>
     <string name="cache_filter_checkboxlist_selectallnone">選擇全部</string>
+    <string name="cache_filter_checkboxlist_add_all_items">Alle Elemente hinzufügen</string>
```

Comment:
Why do we get a German text in zh-rTW language file?

Reply:
That could be my fault. Lately Crowdin links from mail seem to be invalid, leading to an automatic retry and you end up with the language in which you worked on instead of the linked one.

Reply:
Ok, so this will be fixed with next update from Crowdin?

Reply:
I just now corrected in, so it should be fixed with next push.

Reply:
Its fixed by the added commits